12.8-2.3/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM postgres:12.8
+
+ENV PLV8_VERSION=2.3.15 \
+    PLV8_SHASUM="8a05f9d609bb79e47b91ebc03ea63b3f7826fa421a0ee8221ee21581d68cb5ba"
+
+RUN buildDependencies="build-essential \
+    ca-certificates \
+    curl \
+    git-core \
+    python \
+    python3 \
+    gpp \
+    cpp \
+    pkg-config \
+    apt-transport-https \
+    cmake \
+    libc++-dev \
+    libc++abi-dev \
+    postgresql-server-dev-$PG_MAJOR \
+    postgresql-contrib" \
+  && runtimeDependencies="libc++1 \
+    libtinfo5 \
+    libc++abi1" \
+  && apt-get update \
+  && apt-get install -y --no-install-recommends ${buildDependencies} ${runtimeDependencies} \
+  && mkdir -p /tmp/build \
+  && curl -o /tmp/build/v$PLV8_VERSION.tar.gz -SL "https://github.com/plv8/plv8/archive/v${PLV8_VERSION}.tar.gz" \
+  && cd /tmp/build \
+  && echo $PLV8_SHASUM v$PLV8_VERSION.tar.gz | sha256sum -c \
+  && tar -xzf /tmp/build/v$PLV8_VERSION.tar.gz -C /tmp/build/ \
```

Comment:
For commands that include variables like these, make sure to enclose the necessary pieces in double quotes for safety. You might be able to run `shell-check` against this to get more details. E.g.

`RUN tar -xzf "/tmp/build/v$PLV8_VERSION.tar.gz" -C /tmp/build/`